checkout-v3/migration-guide.md

```diff
@@ -12,9 +12,9 @@ Checkout v3 provides a more stream-lined, refined checkout experience for both
 the payer and the merchant. The response models have been reduced in size and
 complexity to simplify integration.
 
-Where Checkout v2 and Payment Instruments were offered more as a mix and match
-package, Checkout v3 is more conveniently divided into four distinct packages:
-_Starter_, _Business_, _Enterprise_, and _Payments Only_.
+Where Checkout v2 and Payment Instruments v1 were offered more as a mix and
+match package, Checkout v3 is more conveniently divided into four distinct
+packages: _Starter_, _Business_, _Enterprise_, and _Payments Only_.
```

Comment:
Should we still talk about Starter & Business?

Reply:
This is a in the migration guide, which is hidden from the documentation and has to be rewritten completely, so I have let it be for now. Will be removed :)